lib/solidus_importer/processors/option_values.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class OptionValues < Base
+      def call(context)
+        @data = context.fetch(:data)
+        return unless option_value_names.any?
+
+        variant = context.fetch(:variant)
+        process_option_values(variant)
+      end
+
+      private
+
+      def process_option_values(variant)
+        option_value_names.each_with_index do |name, i|
+          option_type = option_type(variant.product, i)
+
+          v = Spree::OptionValue.find_or_initialize_by(
+            option_type: option_type,
```

Comment:
I would suggest using a different name for this local variable (`option_type`) to avoid misunderstandings with the method name.